src/main/kotlin/no/nav/syfo/varsel/esyfovarsel/EsyfovarselService.kt

```diff
@@ -26,4 +26,25 @@ class EsyfovarselService(private val producer: EsyfovarselProducer) {
         )
         producer.sendVarselTilEsyfovarsel(esyfovarselHendelse)
     }
+
+    fun ferdigstillSvarMotebehovForArbeidsgiver(narmestelederFnr: String, ansattFnr: String, orgnummer: String) {
+        val esyfovarselHendelse = NarmesteLederHendelse(
+            HendelseType.NL_DIALOGMOTE_SVAR_MOTEBEHOV,
+            VarselData(VarselStatus(ferdigstilt = true)),
```

Comment:
Kunne vi lagt til en tilsvarende `VarselData(VarselStatus(ferdigstilt = false)),` i  `sendSvarMotebehovVarselTilArbeidstaker` og `sendSvarMotebehovVarselTilNarmesteLeder` i stedet for `null`, slik at vi kan vi pakke ut `SM_DIALOGMOTE_SVAR_MOTEBEHOV` og `NL_DIALOGMOTE_SVAR_MOTEBEHOV` på samme måte i `esyfovarsel` ?